src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -40,7 +40,7 @@ object AudioApiProperties extends LazyLogging {
   val SearchServer = propOrElse("SEARCH_SERVER", "http://search-audio-api.ndla-local")
   val SearchRegion = propOrElse("SEARCH_REGION", "eu-central-1")
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
-  val SearchIndex = "audios"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "audios")
```

Comment:
@runesto ElasticSearch-instansen utenfor testklassen er den som kjører i docker lokalt på din maskin. Den har mest sannsynlig allerede en index som heter `audios` dersom man har kjørt opp audio-apiet lokalt. Vi vil i integrasjonstesten gjerne teste mot en annen index enn den som man bruker lokalt, for å ikke rote til `audios`-indeksen. Testen forventer også at et spesifikt antall dokumenter ligger der. Så om man bare bruker indeksene som allerede eksisterer (og med dokumenter som vi ikke har kontroll på), så er det stor sannsynlighet for at testen vil gi uventede resultater og feile.